### PR TITLE
fix: replaced testing Python filetypes with Go, fixed generation of a…

### PR DESCRIPTION
**Pull request by nekhvoya**
… random strings by starting from a letter always

### CODE REVIEW

**Comment by supakeen** [comment]
Ok, two things.

1. Can you add the reason for this change to the PR description (the fact that `pinnwand` does a `strip()` on the incoming text which will remove leading and trailing spaces.
2. `pinnwand` does `.strip()` on the incoming text, it'd probably be better to replicate this in the testcase as right now you're not removing trailing whitespace.

File: test/e2e/utils/string_utils.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 
 def random_string(size=1000) -> str:
-    return (
+    return random.choice(string.ascii_letters) + (
```
Comment (supakeen) [inline test/e2e/utils/string_utils.py:6]: It's better to call `.strip()` on the resulting string as `pinnwand` strips both leading and trailing whitespace.

**Review by supakeen — CHANGES_REQUESTED** [review]
Can you add the reason for this change to the PR description (the fact that `pinnwand` does a `strip()` on the incoming text which will remove leading and trailing spaces and the fact that `pinnwand` auto-indents Python code?

**Comment by supakeen** [comment]
Related to: #237

File: test/e2e/utils/string_utils.py (hunk 1 above)
Comment (nekhvoya) [reply test/e2e/utils/string_utils.py]: done

**Review by supakeen — APPROVED** [review]

**Pull request merged**